backend/controllers/ratings_controller.js

```diff
@@ -99,12 +102,15 @@ export const getRatingsByUser = asyncHandler(async (req, res) => {
       },
       {
         $project: {
+          _id: '$_id',
```

Comment:
Maybe use a more descriptive name for _id. I know it is the ObjectID for a Rating in Mongo, but "_id" would also be the name of the ObjectID for other relations. Not a problem until they overlap (e.g. we use a Rating object id while working with User data.